src/components/settings/SettingsWebcamsTab.vue

```diff
@@ -257,6 +261,7 @@ interface webcamForm {
         'webcam-ipstreamer': Ipstreamer,
         'webcam-webrtc-camerastreamer': WebrtcCameraStreamer,
         'webcam-hlsstreamer': Hlsstreamer,
+        'webcam-janus': JanusStreamer,
```

Comment:
pls rename it to webcam-webrtc-janus (like the webrtc-camerastreamer example two lines above)